### DISCUSSION TITLE
Is it possible to turn off Garmin Varia RTL515 using SBL?

### DISCUSSION THREAD

**Discussion by kregielpl** [maca88/SmartBikeLights#63] [General]
Hi,

I am wondering if it is possible to turn off the Garmin Varia RTL515 using SBL? When I go for a ride, I need around 15-20 minutes to leave town. It would be great if I could turn the Garmin Varia on and off using SBL, because the constant beeping is annoying.

Thanks for your reply!

**Comment by maca88** [comment]
Hi,

Unfortunately, it is not possible to completely turn off (including radar beeping) Varia RTL515 by using a Garmin CIQ application as there is no API for this. In theory, with a CIQ application it would be possible to implement the beeping, so that the CIQ application would be responsible to trigger the beeps when a vehicle is detected and then you would need to keep the native Varia beeping always off. The problem with this is that such functionality is not permitted by Garmin and the application could be removed from the store due to safety issues.
